reform_cage.py

```diff
@@ -0,0 +1,176 @@
+"""
+From a two-component cage .mol file, returns:
+
+	1) The SMILES code of the building blocks.
+	1.1) The active coordination number of the linkers.
+        1.2) The active functional group.
+        1.3) The number of times a building block has been used in a cage.
+
+	2) The topology of the cage.
+
+        3) Optional Arguments.
+        3.1) Optional (-c): Reform the unrelaxed cage from .mol file.
+        3.2) Optional (-b): Reform the unrelaxed building-blocks (bb).
+
+Usage: $ python reform_cage.py <file.mol>
+
+Author: James T. Pegg
+
+"""
+import argparse
+import stk
+from rdkit.Chem import AllChem
+import operator
+import os
+
+def reform_cage(name, result, deconstructed_topology):
+    """
+    Reform the unrelaxed two-component stk cage.
+
+    """
```

Comment:
The docstring should have a Parameters Section and a Returns section.
See 
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html